db/fixtures/miq_policies.yml

```diff
@@ -0,0 +1,29 @@
+---
+- :name: Analyse all new container images for known vulnerabilities with OpenSCAP ruleset
```

Comment:
@jrafanie @Fryguy Can all names be shorter and underscored and the descriptions have the long version? The user only sees the description, the name is used for linking entities (and adding the default policy to new container providers in #7567)